src/bootstrap/src/core/config/config.rs

```diff
@@ -1128,7 +1127,6 @@ define_config! {
         debuginfo_level_tests: Option<DebuginfoLevel> = "debuginfo-level-tests",
         backtrace: Option<bool> = "backtrace",
         incremental: Option<bool> = "incremental",
-        parallel_compiler: Option<bool> = "parallel-compiler",
```

Comment:
We usually keep options for a while with a FIXME message (e.g., FIXME: remove this at the end of the year) in `change_tracker.rs` to avoid causing breaking changes during back-and-forth git checkouts.